extras/cronwrapper

```diff
@@ -51,11 +51,11 @@ if $LOGGING; then
     RET=1
   fi
 else
-  "${SCRIPT}" "${CONF}" 2>&1 >${LOGFILE}
+  "${SCRIPT}" "${CONF}" >${LOGFILE} 2>&1
   RET=$?
 fi
 
-if [ $RET -ne 0 -a $RET -ne 2 -a $RET -ne 5 ]; then
+if [ $RET -ne 2 -a $RET -ne 5 ]; then
```

Comment:
Normally, I'd agree, but I'm trying to match the old -c behavior which was that we would notify if anything was actually done, not just if an actual error came up. We can add a note to the wiki about adding the 0 check back in if you **only** want to be notified on errors. (or vice versa, but I personally prefer to receive an email if an update is applied)